Source/WebCore/css/calc/CSSCalcTree+Parser.cpp

```diff
@@ -627,7 +627,7 @@ static std::optional<TypedChild> consumeAnchor(CSSParserTokenRange& tokens, int
             return sideIdent;
 
         if (tokens.peek().type() == PercentageToken)
-            return tokens.consumeIncludingWhitespace().numericValue();
+            return tokens.consumeIncludingWhitespace().numericValue() / 100;
```

Comment:
Probably make things easier for future us if you throw the rest of the <anchor()> grammar at the top of the function:

```
<anchor-element> = <dashed-ident>
<anchor-side>    = inside | outside | top | left | right | bottom | start | end | self-start | self-end | <percentage> | center
```

Not needed for this change, but I have been trying to wrap raw doubles in a semantic struct when I can. The calc(), Percentage struct is probably not right perfect for this, since there is no percent hint needed, but you can just give that the default value. Alternatively, just define a Anchor::SidePercentage struct.

Reply:
Good point, nesting calc is allowed I believe.